Error in create backup

Hello, 
When calling a backup I get the following error:

`.\proxmoxbackupgo_cli.exe -config .\backup_docs.json `

> Upgraderesp: HTTP/1.1 101 Switching Protocols
connection: upgrade
upgrade: proxmox-backup-protocol-v1
date: Wed, 28 Aug 2024 15:14:13 GMT
Successfully upgraded to HTTP/2.
Downloaded previous DIDX: 24 bytes
Previous index has wrong magic (no valid)!
Known chunks: 0!
Chunk size min is 1048576 , max 16777216
Chunk size min is 1048576 , max 16777216
Writer id:  1
Writer id:  2
New chunk[752831ea001b9be097232df514e4a83c74ef11e4a3adb64c5ca79d2c62b97d8e] 1059307 bytes
Found leftover snapshot, deleting it...
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x1 addr=0x0 pc=0x7ff6f2981645]
goroutine 1 [running, locked to thread]:
sync/atomic.(*Uint64).Add(...)
        C:/Program Files/Go/src/sync/atomic/type.go:183
main.backup.func1({0xc00023a000, 0x10000, 0x10000})
        PATH_TO_SOURCES/proxmoxbackupclient_go/main.go:224 +0x525
main.(*PXARArchive).Flush(0xc000324000)
        PATH_TO_SOURCES/proxmoxbackupclient_go/pxar.go:168 +0x50
main.(*PXARArchive).WriteFile(0xc000324000, {0xc00024c080, 0x80}, {0xc000114fc6, 0xa})
        PATH_TO_SOURCES/proxmoxbackupclient_go/pxar.go:466 +0x433
main.(*PXARArchive).WriteDir(0xc000324000, {0xc000312480, 0x75}, {0x0, 0x0}, 0x1)
        PATH_TO_SOURCES/proxmoxbackupclient_go/pxar.go:302 +0x7bf
main.backup(0xc000142608, 0x0, 0x0, {0x0, 0x0}, {0xc000112040, 0x1d})
        PATH_TO_SOURCES/proxmoxbackupclient_go/main.go:294 +0x674
main.main()
        PATH_TO_SOURCES/proxmoxbackupclient_go/main.go:95 +0x27e

Any ideas?

--- a/main.go
+++ b/main.go
@@ -48,8 +48,8 @@ func (c *ChunkState) Init() {
 }
 
 func main() {
-	var newchunk *atomic.Uint64
-	var reusechunk *atomic.Uint64
+	var newchunk *atomic.Uint64 = new(atomic.Uint64)
+	var reusechunk *atomic.Uint64 = new(atomic.Uint64)
 
 	cfg := loadConfig()
 

